src/EventGridEmulator/EventGridEmulator.csproj

```diff
@@ -2,12 +2,17 @@
   <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
     <DebugType>none</DebugType>
   </PropertyGroup>
+  <PropertyGroup>
+    <UserSecretsId>03056322-03d7-4174-ac77-5a62892a7a87</UserSecretsId>
+    <DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
```

Comment:
Is that really necessary since we specify linux-musl-x64 in the dockerfile? And also what secret did we set with the UserSecretsId?

Reply:
That was added by docker-compose project.